testrunner-running/pom.xml

```diff
@@ -41,6 +41,11 @@
             <groupId>xmlunit</groupId>
             <artifactId>xmlunit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-launcher</artifactId>
+            <version>1.1.0</version>
```

Comment:
Since this dependency is included in parent's dependencyManagement, there is no need to include version here.

Reply:
Done